### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by HeroicEric**
fixes some typos

### CODE REVIEW

File: source/posts/2015-11-18-phoenix-is-not-rails.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Elixir conventions structures your projects as small composable "applications" t
   ```
 [source](https://github.com/phoenixframework/phoenix/blob/9f9c4663b304a3ff885cc8356cad278e100eb499/mix.exs#L28-L38)
 
-2. If a module is specified, the `start/2` function of this module is invoked 
+2. If a module is specified, the `start/2` function of this module is invoked:
```
Comment (HeroicEric) [inline source/posts/2015-11-18-phoenix-is-not-rails.md]: was inconsistent with the previous item in the list

File: source/posts/2015-11-18-phoenix-is-not-rails.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ Elixir conventions structures your projects as small composable "applications" t
   [source](https://github.com/phoenixframework/phoenix/blob/7692aef141f6eab5ad9a0e88875f42c8b02b117d/lib/phoenix.ex#L3  0)
 
 3. The `start/2` function must return the identifier of a supervised process, such as ` Phoenix.Supervisor.start_link` above
+
   [source](https://github.com/phoenixframework/phoenix/blob/7692aef141f6eab5ad9a0e88875f42c8b02b117d/lib/phoenix.ex#L41)
```
Comment (HeroicEric) [inline source/posts/2015-11-18-phoenix-is-not-rails.md]: was inconsistent with the other source links in the list

File: source/posts/2015-11-18-phoenix-is-not-rails.md [hunk 3]
```diff
@@ -207,7 +208,7 @@ defmodule MyApp.Router do
 end
 
 defmodule MyApp.DashboardController do
-  plug AdminAuthentication                       # plugged only on this controller
+  plug AdminAuthentication # plugged only on this controller
```
Comment (HeroicEric) [inline source/posts/2015-11-18-phoenix-is-not-rails.md]: Had to horizontally scroll to read this comment

**Comment by bcardarella** [comment]
Thanks @HeroicEric!

**Pull request merged**